### ISSUE TITLE
findTransitionState should not default to assuming a cluster

### ISSUE THREAD

**Issue by js850** [pele-python/pele#3]
by default it uses orthogopt().  this should be changed.

**Comment by smcantab** [comment]
yes this is dangerous, I think that by default it should assume that there are no symmetries (+ raise a warning). I am happy to change this and send a PR

**Comment by js850** [comment]
please do
